src/AutoPilotPlugins/PX4/PX4TuningComponentPlaneRate.qml

```diff
@@ -0,0 +1,173 @@
+/****************************************************************************
+ *
+ * (c) 2021 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+import QtQuick          2.3
+import QtQuick.Controls 1.2
+import QtQuick.Layouts  1.2
+
+import QGroundControl               1.0
+import QGroundControl.Controls      1.0
+import QGroundControl.FactSystem    1.0
+import QGroundControl.FactControls  1.0
+import QGroundControl.ScreenTools   1.0
+import QGroundControl.Vehicle       1.0
+
+ColumnLayout {
+    width: availableWidth
+    anchors.fill: parent
+    property alias autotuningEnabled: pidTuning.autotuningEnabled
+
+    GridLayout {
+        columns: 2
+    }
+    PIDTuning {
+        width: availableWidth
+        id:    pidTuning
+
+        property var roll: QtObject {
+            property string name: qsTr("Roll")
+            property var plot: [
+                { name: "Response", value: globals.activeVehicle.rollRate.value },
+                { name: "Setpoint", value: globals.activeVehicle.setpoint.rollRate.value }
+            ]
+            property var params: ListModel {
+                ListElement {
+                    title:          qsTr("Porportional gain (FW_RR_P)")
+                    description:    qsTr("Porportional gain.")
+                    param:          "FW_RR_P"
+                    min:            0.0
+                    max:            1
+                    step:           0.005
+                }
+                ListElement {
+                    title:          qsTr("Differential Gain (FW_RR_D)")
+                    description:    qsTr("Damping: increase to reduce overshoots and oscillations, but not higher than really needed.")
+                    param:          "FW_RR_D"
+                    min:            0.0
+                    max:            1.0
+                    step:           0.005
+                }
+                ListElement {
+                    title:          qsTr("Integral Gain (FW_RR_I)")
+                    description:    qsTr("Generally does not need much adjustment, reduce this when seeing slow oscillations.")
+                    param:          "FW_RR_I"
+                    min:            0.0
+                    max:            0.2
```

Comment:
There is an airframe that has set this to 0.5

Reply:
@bkueng But should these limit follow the parameter meta data? https://github.com/PX4/PX4-Autopilot/blob/92fbd86b463dabd4c1ff0e4cec5c4205dec402f5/src/modules/fw_att_control/fw_att_control_params.c#L199-L212